packages/validator/src/commands/convertCmd.ts

```diff
@@ -9,27 +10,52 @@ import { invalidConversionMessage } from '../utils/messages';
 const oas2ois = 'OAS2OIS.json';
 const ois2config = 'OIS2Config.json';
 
-if (process.env.npm_config_template) {
-  console.log(
-    JSON.stringify(convert(process.env.npm_config_specs || process.argv[3], process.env.npm_config_template), null, 2)
-  );
+const args = yargs(hideBin(process.argv))
+  .option('from', {
+    description: 'Name of the source airnode specification format',
+    demandOption: 'to',
+    default: '',
+    string: true,
+  })
+  .option('to', {
+    description: 'Name of the target airnode specification format',
+    demandOption: 'from',
+    default: '',
+    string: true,
+  })
+  .option('template', {
+    description: 'Path to validator template file',
+    alias: 't',
+    default: '',
+    string: true,
+  })
+  .option('specification', {
+    description: 'Path to specification file that will be validated',
+    default: '',
+    alias: ['specs', 's'],
+    string: true,
+  })
+  .option('specs_only', {
+    description: 'Instead of standard validator output, only the result of conversion will be returned',
+  }).argv;
+
+if (args.template) {
+  console.log(JSON.stringify(convert(args.specification, args.template), null, 2));
 } else {
-  const from = (process.env.npm_config_from || process.argv[2] || '').toLowerCase();
-  const to = (process.env.npm_config_to || process.argv[3] || '').toLowerCase();
-  const specs = process.env.npm_config_specs || process.argv[4];
-  const version = process.env.npm_config_fromVersion || process.argv[5];
   const messages: Log[] = [];
   let res: Result = { valid: false, messages: [], output: {} };
+  args.from = args.from.toLowerCase();
+  args.to = args.to.toLowerCase();
 
-  if (!from || !to) {
+  if (!args['from'] || !args['to']) {
     res.messages.push(logger.error('Conversion source and target specification must be provided'));
-  } else if (from === 'oas' && to === 'ois') {
-    res = convert(specs, utils.getPath(oas2ois, messages, version));
+  } else if (args.from === 'oas' && args.to === 'ois') {
```

Comment:
Why do you sometime access the field with `[]` and sometime with `.`? I don't see a reason to do that here.

Reply:
Updated in da523b323077b3a7c9a8b396bfd8640243d2c9d1